portfolio/src/main/java/com/google/sps/servlets/RandomFactServlet.java

```diff
@@ -28,13 +29,14 @@
 public class RandomFactServlet extends HttpServlet {
 
   private final String CONTENT_TYPE = "text/html;";
-  private List<String> facts =
-      Arrays.asList(
-          new String[] {
-            "Our air is composed of mostly nitrogen.",
-            "Moore's Law is an observation.",
-            "The Summit supercomputer can reach up to 200 petaFLOPS."
-          });
+  private final List<String> facts =
+      Collections.unmodifiableList(
+          Arrays.asList(
+              new String[] {
```

Comment:
Remove the object array. Please look at the javadoc for this method.
https://docs.oracle.com/javase/8/docs/api/index.html?java/util/List.html